File: SocialProWeb/src/SocialPro/UserBundle/Entity/Tache.php
<?php

namespace SocialPro\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use SocialPro\UserBundle\Entity\Competence;

/**
 * Tache
 *
 * @ORM\Table(name="tache", indexes={@ORM\Index(name="idCompetence", columns={"id_competence"})})
 * @ORM\Entity
 */
class Tache
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_tache", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idTache;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="text", length=65535, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text", length=65535, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="complexite", type="text", length=65535, nullable=false)
     */
    private $complexite;

    /**
     * @var Competence
     *
     * @ORM\ManyToOne(targetEntity="Competence")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_competence", referencedColumnName="id_competence")
     * })
     */
    private $idCompetence;

    /**
     * @var \Doctrine\Common\Collections\Collection
     *
     * @ORM\ManyToMany(targetEntity="User", mappedBy="idTache")
     */
    private $idUser;

    /**
     * Constructor
     */
    public function __construct()
    {
        $this->idUser = new \Doctrine\Common\Collections\ArrayCollection();
    }


    /**
     * Get idTache
     *
     * @return integer
     */
    public function getIdTache()
    {
        return $this->idTache;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Tache
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Tache
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set complexite
     *
     * @param string $complexite
     *
     * @return Tache
     */
    public function setComplexite($complexite)
    {
        $this->complexite = $complexite;

        return $this;
    }

    /**
     * Get complexite
     *
     * @return string
     */
    public function getComplexite()
    {
        return $this->complexite;
    }

    /**
     * Set idCompetence
     *
     * @param \SocialPro\UserBundle\Entity\Competence $idCompetence
     *
     * @return Tache
     */
    public function setIdCompetence(\SocialPro\UserBundle\Entity\Competence $idCompetence = null)
    {
        $this->idCompetence = $idCompetence;

        return $this;
    }

    /**
     * Get idCompetence
     *
     * @return \SocialPro\UserBundle\Entity\Competence
     */
    public function getIdCompetence()
    {
        return $this->idCompetence;
    }

    /**
     * Add idUser
     *
     * @param \SocialPro\UserBundle\Entity\User $idUser
     *
     * @return Tache
     */
    public function addIdUser(\SocialPro\UserBundle\Entity\User $idUser)
    {
        $this->idUser[] = $idUser;

        return $this;
    }

    /**
     * Remove idUser
     *
     * @param \SocialPro\UserBundle\Entity\User $idUser
     */
    public function removeIdUser(\SocialPro\UserBundle\Entity\User $idUser)
    {
        $this->idUser->removeElement($idUser);
    }

    /**
     * Get idUser
     *
     * @return \Doctrine\Common\Collections\Collection
     */
    public function getIdUser()
    {
        return $this->idUser;
    }
}

File: SocialProMobile/src/Entity/Tache.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Entity;

/**
 *
 * @author bechir23
 */
public class Tache {
    private int id_tache;
    private String nom;
    private String description;
    private String complexite;

    public Tache(int id_tache, String nom, String description, String complexite) {
        this.id_tache = id_tache;
        this.nom = nom;
        this.description = description;
        this.complexite = complexite;
    }

    public Tache() {
    }

     public int getId_tache() {
        return id_tache;
    }
    public String getNom() {
        return nom;
    }
    

    public String getDescription() {
        return description;
    }
    
     public String getComplexite() {
        return complexite;
    }

    
    public void setId_tache(int id_tache) {
        this.id_tache = id_tache;
    }
    public void setNom(String nom) {
        this.nom = nom;
    }
   
    public void setDescription(String description) {
        this.description = description;
    }
    
     public void setComplexite(String complexite) {
        this.complexite = complexite;
    }


    

    @Override
    public String toString() {
        return "Tache{" + "id_tache=" + id_tache + ", nom=" + nom + ", description=" + description + ", complexite=" + complexite + '}';
    }

  
    
    
    
    
          
}
